conformance/tests/httproute-matching-across-routes.go

```diff
@@ -116,10 +116,23 @@ var HTTPRouteMatchingAcrossRoutes = suite.ConformanceTest{
 }
 
 func testName(tc http.ExpectedResponse, i int) string {
```

Comment:
I think this function should probably be moved. Since it's dependent on `http.ExpectedResponse`, we could move it to the file that defines that and make a function like this:


```
func (er *ExpectedResponse) TestCaseName (num int) {
   ...
}
```

Reply:
Moved the function to utils/http/http.go